packages/credential-w3c/src/__tests__/issue-verify-flow.test.ts

```diff
@@ -18,6 +18,11 @@ import { EthrDIDProvider } from '../../../did-provider-ethr/src'
 import { ContextDoc } from '../../../credential-ld/src/types'
 import { Resolver } from 'did-resolver'
 import { getResolver as ethrDidResolver } from 'ethr-did-resolver'
+import { CredentialIssuerLD } from '../../../credential-ld/src/action-handler'
+import { LdDefaultContexts } from '../../../credential-ld/src/ld-default-contexts'
+import { VeramoEd25519Signature2018 } from '../../../credential-ld/src/suites/Ed25519Signature2018'
+import { VeramoEcdsaSecp256k1RecoverySignature2020 } from '../../../credential-ld/src/suites/EcdsaSecp256k1RecoverySignature2020'
+import { VerifiableCredential } from '@veramo/core'
```

Comment:
```suggestion
import { VerifiableCredential } from '../../../core/src/'
```

We have to use relative paths in tests to test against local source code instead of the packages from NPM